### DISCUSSION TITLE
Insufficient Documentation

### DISCUSSION THREAD

**Discussion by lortschi** [JustFly1984/react-google-maps-api#3253] [Ideas]
Dear Team / Contributors,

I struggled a lot regarding lack of many property and entitiy documentations in this version of react google maps api. Few times I thought throw it to the bin and let's do on my own the react implementation of vanilla js google api or to use the old outdated react-google-maps repo: https://github.com/tomchentw/react-google-maps. 

Please do for all developers a favour and enrich the documentations to avoid lot of stackoverflow researching.

Just here is an example of documentation lacked property: "zoomControlOptions". 

No where is it documented how to setup properly. The docs shows with nested:
"position": google.maps.ControlPosition.RIGHT_CENTER 
....this won't work because google is undefined. And what is about of RIGHT_CENTER? Where is this constant defined? This is actually a number of position that is no where described. 

etc etc ...

**Comment by lovedeep5** [comment]
You can find "google" in your window object, if where you you find reference to google add window as prefix.
